app/src/test/java/com/puzzlebench/clean_marvel_kotlin/presentation/mvp/CharactersPresenterTest.kt

```diff
@@ -33,15 +35,15 @@ class CharacterPresenterTest {
 
         getCharacterServiceUseCase = GetCharacterServiceUseCase(characterServiceImp)
         val subscriptions = mock(CompositeDisposable::class.java)
-        characterPresenter = CharacterPresenter(view, getCharacterServiceUseCase, subscriptions)
+        charactersPresenter = CharactersPresenter(view, getCharacterServiceUseCase, subscriptions)
 
 
     }
 
     @Ignore
     fun reposeWithError() {
         Mockito.`when`(getCharacterServiceUseCase.invoke()).thenReturn(Observable.error(Exception("")))
```

Comment:
Here you have a hardcode, please avoid it.

Reply:
Done